<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADM_atendimentos</class>
 <widget class="QMainWindow" name="ADM_atendimentos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="botaoInicio">
        <property name="styleSheet">
         <string notr="true">QPushButton { 
color: rgb(68, 68, 68);
background-color: rgb(255, 255, 255);
border-color: rgb(48, 245, 255);
}

QPushButton:pressed { 
color: rgb(68, 68, 68);
background-color: rgb(170, 255, 255);
border-color: rgb(48, 245, 255);
}</string>
        </property>
        <property name="text">
         <string>Início</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botaoAtendimentos">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(144, 144, 144);
background-color: rgb(176, 251, 255);
border-color: rgb(48, 245, 255);
</string>
        </property>
        <property name="text">
         <string>Atendimentos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botaoCadastros">
        <property name="styleSheet">
         <string notr="true">QPushButton { 
color: rgb(68, 68, 68);
background-color: rgb(255, 255, 255);
border-color: rgb(48, 245, 255);
}

QPushButton:pressed { 
color: rgb(68, 68, 68);
background-color: rgb(170, 255, 255);
border-color: rgb(48, 245, 255);
}</string>
        </property>
        <property name="text">
         <string>Cadastros</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botaoMaterial">
        <property name="styleSheet">
         <string notr="true">QPushButton { 
color: rgb(68, 68, 68);
background-color: rgb(255, 255, 255);
border-color: rgb(48, 245, 255);
}

QPushButton:pressed { 
color: rgb(68, 68, 68);
background-color: rgb(170, 255, 255);
border-color: rgb(48, 245, 255);
}</string>
        </property>
        <property name="text">
         <string>Material</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botaoSair">
        <property name="styleSheet">
         <string notr="true">QPushButton { 
color: rgb(68, 68, 68);
background-color: rgb(255, 255, 255);
border-color: rgb(48, 245, 255);
}

QPushButton:pressed { 
color: rgb(68, 68, 68);
background-color: rgb(170, 255, 255);
border-color: rgb(48, 245, 255);
}</string>
        </property>
        <property name="text">
         <string>Sair</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="atendimentos">
       <attribute name="title">
        <string>Atendimentos marcados</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>2</y>
          <width>781</width>
          <height>481</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_pesquisa">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Pesquise o tipo de tendimento e o dia:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QRadioButton" name="rdbotaoEscolher_Todos">
             <property name="text">
              <string>Todos</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rdbotaoEscolher_Consulta">
             <property name="text">
              <string>Consultas</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rdbotaoEscolher_Cirurgia">
             <property name="text">
              <string>Cirurgias</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="botaoPesquisar">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(7, 255, 226);
border-color: rgb(48, 245, 255);
</string>
             </property>
             <property name="text">
              <string>Pesquisar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCalendarWidget" name="calendario_marcados"/>
         </item>
         <item>
          <widget class="QTableWidget" name="tw_atendimentos"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="botaoExcluir">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 126, 126);
background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Excluir</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Marcar">
       <attribute name="title">
        <string>Marcar um atendimento</string>
       </attribute>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>781</width>
          <height>481</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="2" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QRadioButton" name="rdbotaoConsulta">
               <property name="text">
                <string>Consulta</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rdbotaoCirurgia">
               <property name="text">
                <string>Cirurgia</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_pacientes">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Selecione o paciente:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tw_pacientes"/>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_medico">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Selecione o médico:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tw_medicos"/>
           </item>
          </layout>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_dia_hora">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Selecione a data:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCalendarWidget" name="calndario">
               <property name="minimumDate">
                <date>
                 <year>2024</year>
                 <month>12</month>
                 <day>1</day>
                </date>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Horário:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QTimeEdit" name="time_edit">
                 <property name="displayFormat">
                  <string>HH</string>
                 </property>
                 <property name="calendarPopup">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="botaoMarcar">
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(7, 255, 226);
border-color: rgb(48, 245, 255);
</string>
               </property>
               <property name="text">
                <string>Marcar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
